config/kafka-monitor.properties

```diff
@@ -39,6 +39,9 @@
     "zookeeper.connect": "localhost:2181",
     "bootstrap.servers": "localhost:9092",
     "produce.record.delay.ms": 100,
+    "produce.topic.autoTopicCreationEnabled": true,
```

Comment:
Ah now I understand your question. Right, kafka-monitor.properties is not used by BasicEndToEndTest.java. It is only used by ./bin/kafka-monitor-start.sh  and KafkaMonitor.java as input.